docs/BR.md

```diff
@@ -1008,6 +1042,83 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
 
 CAs MUST document potential issuances that were prevented by a CAA record in sufficient detail to provide feedback to the CAB Forum on the circumstances, and SHOULD dispatch reports of such issuance requests to the contact(s) stipulated in the CAA iodef record(s), if present. CAs are not expected to support URL schemes in the iodef record other than mailto: or https:.
 
+#### 3.2.2.9 Multi-Perspective Domain Validation
+
+Multi-Perspective Domain Validation attempts to corroborate the Primary Domain Validation Determination and Primary CAA Determination by re-completing the entirety of the domain validation process from multiple geographic Network Perspectives before subscriber certificate issuance. This process can offer improved protection against equally-specific prefix attacks (i.e., a form of Border Gateway Protocol hijack).
+
+Network Perspectives MUST be spread across a minimum of two distinct continents. Each Network Perspective MUST independently complete the entirety of:
```

Comment:
Hi Aaron, based on my interpretation, after March 15, 2025 this line is indeed redundant with 1062. However, before March 15 2025, line 1062 does not take effect (because before March 15 2025 a CA can proceed with issuance regardless of the result of MPDV). Thus, this line (1049) would require the spread of network perspectives across two distinct continents even in logging only mode. Personally I am still OK removing it for the sake of simplicity of the draft, but I did want to point out that I feel it would have a slight change in the interpretation of the requirement.

Reply:
Good catch. But I concur that making the change is good: encourage people to set up MPDV in multiple locations right off the bat.